spec/queue.js

```diff
@@ -0,0 +1,60 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+
+describe.only('Queue', () => {
```

Comment:
Add explicit tests for `front()` and `back()` (as well as their expected behavior when the queue is empty.